CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ v4.2
 - Fix a segfault that occurs when using OpenSim's Python Package with Anaconda's Python on a Mac.
 - Expose PropertyHelper class to python bindings to allow editing of objects using the properties interface (useful for editing objects defined in plugins) in python (consistent with Java/Matlab).
 - Whitespace is trimmed when reading table metadata for STO, MOT, and CSV files.
+- Fix OpenSense calibration algorithm to handle models facing arbitrary direction, and handle angle sign correctly in all scenarios.
```

Comment:
The point of the heading correction was to align the gait model so that begins facing the positive X-direction (that is where the pelvis faces when pelvis_rotation is zero)  That is to enable the experimental IMU to be facing any direction, but when represented in OpenSim it will start walking in the X-direction. Is it now going to face the direction determined by the initial pelvis_rotation? What does that mean? How do I now compare pelvis_rotations between subjects?